### ISSUE TITLE
Movies / Series: What do I have already seen?

### ISSUE THREAD

**Issue by ochurlaud** [macaw-movies/macaw-movies#132]
People I asked told me that an important feature would be that Macaw remembers what they have seen. So that they don't have to remember "I'm watching How I met your mother, I'm season 73, episode 12".

So it means that all movies should go in the ToWatch playlist at the importation, the playlist should update itself each time we open a movie, and easily come back to its previous state (in case you open it and don't finish or change your mind).

Another playlist "What I want to see in the next few days" may be added to replace the current "ToWatch".

Discussion is opened to know how we design this.

**Comment by ochurlaud** [comment]
Use QProcess. On start, launch a timer, catch the signal exited/finished and read the timer. If the time is bigger than... then consider as watched.

**Comment by sebtouze** [comment]
Idea seems good but regarding the timer I think it will not work. If user put movie in pause for lunch at his / her come back timer will be greater that movie duration but film not finished. 

VLC has a function to play at last stop on Android. There should be a way of doing this on computer too. Maybe a specific exit code.

**Comment by ochurlaud** [comment]
Yes but we don't work with one or 2 media players... And it would be specific code for each
